### ISSUE TITLE
Possible extension?

### ISSUE THREAD

**Issue by xxing21** [mikeizbicki/cmc-csci181#14]
I am wondering if it is possible to have an extension for hw7. I suggest Saturday midnight?
Thank you!!

**Comment by mikeizbicki** [comment]
Yes, I will grant an extension for everyone until Saturday 11 Apr @ midnight.
